interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK-ratls.md

```diff
@@ -0,0 +1,158 @@
+Testing between Gramine and rats-tls
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGXSDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
```

Comment:
Why do you call it `SGXSDK` (in one word)? Please fix it, the official name is `SGX SDK` (in two words).

Fix it here and everywhere in this document.

Reply:
updated.